packages/next/next-server/server/config.ts

```diff
@@ -99,7 +99,12 @@ function assignDefaults(userConfig: { [key: string]: any }) {
     }
   })
 
-  return { ...defaultConfig, ...userConfig }
+  const result = { ...defaultConfig, ...userConfig }
+  if (result.experimental && result.experimental.css) {
+    // The new CSS support requires granular chunks be enabled.
+    result.experimental.granularChunks = true
+  }
```

Comment:
Should we log that we are enabling this?

Reply:
I don't feel strongly either way. It's all experimental and chunking is going on-by-default soon.

Reply:
Discussed in Slack